pkg/sfu/pacer/base.go

```diff
@@ -44,14 +49,44 @@ func (b *Base) SetInterval(_interval time.Duration) {
 func (b *Base) SetBitrate(_bitrate int) {
 }
 
+// prepare adds given extensions,
+// for extensions that need to be added with timing as close to packet
+// send (abs-send-time and transport-cc), a dummy extension is used
+// so that header size can be calculated.
+func (b *Base) Prepare(p *Packet) (int, int) {
+	// clear out extensions that may have been in the forwarded header
+	p.Header.Extension = false
+	p.Header.ExtensionProfile = 0
+	p.Header.Extensions = []rtp.Extension{}
```

Comment:
can we reuse the extensions slice?

Reply:
Good point. Let me change that. No need to define another extension type here. Will use the RTP one directly.

Reply:
oh the `rtp.Extension` fields are private. So, can't use it directly. I think I am misunderstanding your question. Can you please elaborate on the re-use you are thinking about?

Reply:
Going to make a change to just do the `SetExtension` calls on RTP header from `downtrack.go`. Not necessary for other extensions to given to the pacer. Pacer can just do the two needed for BWE.

Reply:
Actually, can do the dummy thing in `downtrack.go` too. Will clean up so that pacer is a bit cleaner.